src/components/FormDrawer.tsx

```diff
@@ -59,18 +58,19 @@ class BaseFormDrawer extends Component<IProps> {
   }
 
   public render () {
-    const { form, isVisible, model, title } = this.props;
+    const { form, isVisible, model, title, width } = this.props;
 
     return (
       <Drawer
+        className='mfa-form-drawer'
```

Comment:
`mfa` --> `Mighty Fields Ant` 🤷‍♂️ 
Might change ? might stay ? Good for now ?

Reply:
Good for now!